### PR TITLE
Search header design

### PR DESCRIPTION
**Pull request by lionel-alves**

### CODE REVIEW

File: weTranslate/Color.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+//
+//  Created by Lionel on 2/21/16.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import UIKit
+
+final class Color {
+
+    static let brand = UIColor(red: 0.961, green: 0.314, blue: 0.235, alpha: 1)
```
Comment (jenniferlim) [inline weTranslate/Color.swift]: Could you change those values by / R/255, G/255, B/255

File: weTranslate/SearchHeaderView.swift [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    }()
+
+    private lazy var fromLanguageButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setTitle("English", forState: .Normal)
+        button.setTitleColor(UIColor.whiteColor(), forState: .Normal)
+        button.addTarget(self, action: "selectFromLanguage:", forControlEvents: .TouchUpInside)
+        return button
+    }()
```
Comment (jenniferlim) [inline weTranslate/SearchHeaderView.swift:32]: What about creating a generic button ?
https://github.com/jenniferlim/weTranslate/issues/8

File: weTranslate/SearchHeaderView.swift [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        stackView.spacing = 10
+        return stackView
+    }()
+
+    private let bodyView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .Vertical
+        stackView.spacing = 25
+        stackView.layoutMargins = UIEdgeInsets(top: 25, left: 25, bottom: 25, right: 25)
```
Comment (jenniferlim) [inline weTranslate/SearchHeaderView.swift:71]: what about creating a constant for 25

File: weTranslate/SearchHeaderView.swift [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+    required init(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+
+    // MARK: - Methods
+
+    func selectFromLanguage(sender: UIButton?) {
+        UIView.animateWithDuration(0.3) {
```
Comment (jenniferlim) [inline weTranslate/SearchHeaderView.swift]: What about creating a constant for 0.3 that could be use in the animation display/hide

File: weTranslate/SearchHeaderView.swift (hunk 2 above)
Comment (lionel-alves) [inline weTranslate/SearchHeaderView.swift:32]: Agreed, let's not block this PR for that though.

**Comment by lionel-alves** [comment]
:8ball:

**Comment by jenniferlim** [comment]
:bowtie:

**Pull request merged**